app/models/project_metadata.rb

```diff
@@ -50,13 +50,13 @@ def project_timestamps
         timestamps = {}
         if project.metadata[:created_by].nil?
           timestamps[:created_by] = current_user.uid
-          timestamps[:created_on] = DateTime.now.strftime("%d-%b-%Y %H:%M:%S")
+          timestamps[:created_on] = Time.current.in_time_zone("America/New_York").iso8601
 
         else
           timestamps[:created_by] = project.metadata[:created_by]
           timestamps[:created_on] = project.metadata[:created_on]
           timestamps[:updated_by] = current_user.uid
-          timestamps[:updated_on] = DateTime.now.strftime("%d-%b-%Y %H:%M:%S")
+          timestamps[:updated_on] = Time.current.in_time_zone("America/New_York").iso8601
```

Comment:
Note: When writing to mediaflux, I think dates have to be in a specific format which is different from this. If it in an unaccepted format— instead of converting to the appropriate format — it will override the created/updated on fields to nil.